modules/system-info/reporters/experiments.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace Elementor\Modules\System_Info\Reporters;
+
+use Elementor\Plugin;
+
+/**
+ * Elementor experiments report.
+ *
+ * Elementor experiment report handler class responsible for generating a report for
+ * the experiments included in Elementor and their status.
+ */
+class Experiments extends Base {
+
+	/**
+	 * Get experiments reporter title.
+	 *
+	 * @return string Reporter title.
+	 */
+	public function get_title() {
+		return 'Elementor Experiments';
+	}
+
+	/**
+	 * Get experiments report fields.
+	 *
+	 * @return array Required report fields with field ID and field label.
+	 */
+	public function get_fields() {
+		return [
+			'experiments' => '',
+		];
+	}
+
+	public function is_tracked() {
+		// Since manually handled by `get_settings_experiments_usage`.
+		return false;
+	}
+
+	/**
+	 * Get Experiments.
+	 *
+	 * Retrieve the list of Elementor experiments and each experiment's status (active/inactive), in HTML table format.
+	 *
+	 * @return array
+	 */
+	public function get_experiments() {
+		$experiments = Plugin::$instance->experiments->get_features();
+
+		$output = '';
+
+		foreach ( $experiments as $experiment ) {
+			// If the state is default, add the default state to the string.
+			$state = Plugin::$instance->experiments->get_feature_state_label( $experiment );
+
+			$output .= '<tr><td>' . $experiment['title'] . ': </td>';
```

Comment:
It should be escaped?
```suggestion
			$output .= '<tr><td>' . esc_html( $experiment['title'] ) . ': </td>';
```

Reply:
done.